packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -197,7 +197,7 @@ contract GoldToken is Initializable, CalledByVm, Freezable, IERC20, ICeloToken {
    * @param value The amount of Celo Gold to transfer.
    * @return True if the transaction succeeds.
    */
-  function _transfer(address to, uint256 value) internal onlyWhenNotFrozen returns (bool) {
+  function _transfer(address to, uint256 value) internal returns (bool) {
```

Comment:
Nit: I think it might be cleaner to leave the modifier here, but remove it from `_transfer`'s callers.